atlas/packages/components/.storybook/theme.js

```diff
@@ -0,0 +1,44 @@
+import { create } from "@storybook/theming/create";
+/* Can't import the theme directly here because it is written in TS.
```

Comment:
Are you sure that's the case? I've managed to import the theme no problem.

One more thing I ended up doing was wrapping the story in `Layout` component from the `app` package. Not sure if that's an anti-pattern with how our code is structured but that enabled no code duplication and one-to-one styling with the app. Potentially we could move the `globalStyles` to `components` package and alleviate this problem. But then again, I don't think this is in the scope of this PR.

Reply:
Just checked, and looks like I can import fine too, not sure what was the issue I had. Nonetheless, I though about it and I think it's better to keep it self contained, so we can reuse it as is for the website Storybook and for future projects.